assemblies/platform/pentaho-cdf-dd/src/main/resources/settings.xml

```diff
@@ -54,4 +54,7 @@
 
   <!-- A flag to disable reflected XSS prevention for the unpredicted cases -->
   <parameter-xss-escaping>true</parameter-xss-escaping>
+
+  <!-- A flag to disable save report, this disables API and user interface. Check PPP-4798 for more info  -->
+  <allow-save-report>true</allow-save-report>

```

Comment:
The name of this flag does not look appropriate given that what is being saved is a CDE dashboard, not a report. Something like `allow-save-dashboard` would do.